templates/errors/404.html

```diff
@@ -2,9 +2,18 @@
 
 {% set page_title = _("Page not found") + " - " + _("Office for National Statistics") %}
 
+{% if schema_theme == "census-nisra" %}
+  {% set contact_us_url = "https://census.gov.uk/ni/contact-us/" %}
+{% elif language_code == "cy" %}
+  {% set contact_us_url = "https://cyfrifiad.gov.uk/cysylltu-a-ni/" %}
+{% else %}
+  {% set contact_us_url = "https://census.gov.uk/contact-us/" %}
+{% endif %}
+
```

Comment:
Ive changed my if statement around this now because of a comment Ian left on my PR [heres](https://github.com/ONSdigital/eq-survey-runner/pull/2384/commits/f841862f1d282f2226981c93a0d0dbdf69c039a1) my commit with the change. He suggested it to set a default before the if statement probably good to keep it consistent.